File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/Lingkaran.java
package id.ac.uph.pbo.pertemuan01;

public class Lingkaran extends BangunDatar {
    private double radius;
    public Lingkaran(double radius) {
        super("Lingkaran");
        this.radius = radius;
    }
    @Override public double hitungLuas()     { return Math.PI * radius * radius; }
    @Override public double hitungKeliling() { return 2 * Math.PI * radius; }
}

File: PYTHON/Pertemuan 01/Lingkaran.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
import math
from BangunDatar import BangunDatar

class Lingkaran(BangunDatar):
    def __init__(self, radius: float):
        super().__init__("Lingkaran")
        self.radius = radius

    def hitung_luas(self) -> float:
        return math.pi * self.radius * self.radius

    def hitung_keliling(self) -> float:
        return 2 * math.pi * self.radius

